docs/Overview.rst

```diff
@@ -36,17 +36,67 @@ MVS Validation Plan
+*	Comparison to other validated model, which replicates one pilot project into a validated optimization model with different component representation to identify the similarities and differences in results; and
   
 *	Sensitivity analysis, through which input-output transformations are studied to show the impact of changing the values of some input parameters.
 
+Implemented Tests
+#################
+
+Unit tests and integration tests are gauged by using test coverage measurement. The MVS covers so far 80% of the modules and sub-modules as seen in the next figure. Since those tests are automated, this coverage is updated for any changes in the model. The figure also shows that the build is currently passing.
+
+.. image:: images/Test_coverage.png
+ :width: 200
+
+A benchmark is a point of reference against which results are compared to assess the operational validity of a model. Benchmark tests are not automated like unit and integration tests, hence it is necessary to check that they are passing for any implemented changes in the model. The following table lists the implemented benchmark tests which cover several features and functionalities of the MVS.
+
+.. list-table:: Benchmark Tests
+   :widths: 25 25
+   :header-rows: 1
+
+   * - Benchmark Test
+     - Expected Result
+   * - Electrical Grid + PV
+     - Maximum use of PV to serve the demand and the rest is compensated from the grid
+   * - Electrical Grid + PV + Battery
+     - Reduced excess energy compared to Grid + PV scenario to charge the battery
+   * - Electrical Grid + Diesel Generator
+     - The diesel generator is only used if its LCOE is less than the grid price
+   * - Electrical Grid + Battery
+     - The grid is only used to feed the load
+   * - Electrical Grid + Battery + Peak Demand Pricing
+     - Battery is charged at times of peak demand and used when demand is larger
+   * - Electrical Grid (Price as Time Series) + Heat Pump + Heat Grid
+     - Heat pump is used when electricity_price/COP is less than the heat grid price
+     
+More tests can still be implemented with regard to the investment model within the MVS. Also, components with two input sources can also be tested.
+
+For sensitivity analysis, the behaviour of the MVS is studied by testing the effect of changing the value of the feed-in tariff for a fixed value of an asset's LCOE such that LCOE_ASSET is less than the electricity price. More input-output transformations can be investigated such as checking the randomness of supply between the electrical grid and a diesel generator when fuel_price/generator_efficiency is equal to electricity_price/transformer_efficiency. Another sensitivity analysis case could be used to know if a diesel generator actually replaces the consumption from the grid at times of peak demand--i.e., dispatch_price is less or equal to peak_demand_charge. The table below lists the implemented sensitivity analysis tests.
```

Comment:
A subsection head would be good. Please add future sensitivity analysis as a bulleted list.

Reply:
It would be nice if the actual graph of this test would be shown here.